### PR TITLE
TAJO-1526: Investigate the frequent failures of Travis CI

### PR DESCRIPTION
**Pull request by jihoonson**
This PR is to find the origin of the problem.

### CODE REVIEW

**Comment by jinossy** [comment]
+1

**Comment by jihoonson** [comment]
Thanks @jinossy. 

The problem seems the memory overflow, especially related to the permanent generation space.
So, I increased the ```permSize``` from 128 MB to 152 MB in this patch.
Additionally, I reduced the number of test tuples  of TestExternalSortExec from 3,000,000 to 100,000. 
If there are any objections, I'll commit soon.

File: tajo-core/src/test/java/org/apache/tajo/engine/planner/physical/TestExternalSortExec.java [hunk 1]
```diff
@@ -61,7 +61,7 @@
   private LogicalPlanner planner;
   private Path testDir;
 
-  private final int numTuple = 3000000;
```
Comment (navis) [inline tajo-core/src/test/java/org/apache/tajo/engine/planner/physical/TestExternalSortExec.java]: This is fixed for proper test of ExternalSortExec by request (It's not using merge sort but memory scanner). I'll fix that in a following issue.

**Comment by jihoonson** [comment]
Right. So, I also decreased the sort buffer size to test external sort.

**Comment by navis** [comment]
ah, good.

**Pull request closed without merging**